browser/resources/settings/email_aliases_page/features.cc

```diff
@@ -0,0 +1,12 @@
+// Copyright (c) 2025 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// You can obtain one at https://mozilla.org/MPL/2.0/.
+
+#include "brave/browser/resources/settings/email_aliases_page/features.h"
+
+namespace email_aliases {
+
+BASE_FEATURE(kEmailAliases, "EmailAliases", base::FEATURE_ENABLED_BY_DEFAULT);
```

Comment:
this should be disabled by default for now

Reply:
fixed

Reply:
I think you're missing changes here because there are several things that you have said are fixed, but I don't actually see any changes here. For something like this github will normally auto-resolve it. If you've just fixed it locally I would wait to comment until you have actually pushed the changes, otherwise it's a bit confusing

Reply:
Sorry for the confusion -- I had fixed things locally and was keeping track. I will now push the changes I have made so far and then push future commits individually so the "fixed" comments are consistent with what's in the PR